local-build.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+# This script builds the gnome runtime snap using a local SDK snap.
+
+import sys
+import os
+
+base_config = 'snap/snapcraft.yaml'
+modified_config = 'snapcraft.yaml'
+sdk_file = 'gnome-46-2404-sdk.snap'
+
+# If there is a modified snapcraft.yaml, delete it before re-generating it
+if os.path.exists(f'./{modified_config}'):
+    os.remove(f'./{modified_config}')
+
+if not os.path.exists(f'./{sdk_file}'):
+    print(f'There is no valid "{sdk_file}" file. Aborting.')
```

Comment:
```suggestion
    print(f'There is no valid "{sdk_file}" file. Aborting.', file=sys.stderr)
```

Or similar, just writing by memory now :)

Reply:
Yes, that's how it's done.